src/lighteval/models/nanotron/nanotron_model.py

```diff
@@ -455,13 +455,10 @@ def loglikelihood(self, requests: List[LoglikelihoodRequest], override_bs=None)
 
         return self._loglikelihood_tokens(
             requests,
-            override_bs=override_bs,
             disable_tqdm=bool(dist.get_rank(self.parallel_context.world_pg) != 0),
         )
 
-    def loglikelihood_rolling(
-        self, requests: List[LoglikelihoodRollingRequest], override_bs: int = 0
-    ) -> List[LoglikelihoodResponse]:
+    def loglikelihood_rolling(self, requests: List[LoglikelihoodRollingRequest]) -> List[LoglikelihoodResponse]:
```

Comment:
same, you could add a kwargs arg to capture it if accidentally passed and avoid failing at this stage

Reply:
Nathan has changed api of the base model at some point. See: https://github.com/huggingface/lighteval/blob/ce1dbb5eab6edf2c8a8c297952e97ea3b3067041/src/lighteval/models/abstract_model.py#L124, so it's correct like this